shared-module/displayio/ColorConverter.c

```diff
@@ -128,9 +129,29 @@ bool common_hal_displayio_colorconverter_get_dither(displayio_colorconverter_t*
     return self->dither;
 }
 
+void common_hal_displayio_colorconverter_make_transparent(displayio_colorconverter_t* self, uint32_t transparent_color) {
+    if (self->transparent_color) {
+        mp_raise_RuntimeError(translate("transparent_color value is already set"));
+    }
+    self->transparent_color = transparent_color;
+}
+
+void common_hal_displayio_colorconverter_make_opaque(displayio_colorconverter_t* self, uint32_t transparent_color) {
+    if (self->transparent_color != transparent_color) {
+        mp_raise_RuntimeError(translate("transparent_color value is not transparent"));
+    }
```

Comment:
While this is nice, I think it's ok to accept anything because all will be opaque after a successful call. This will save a bit of space.

```suggestion
```

Reply:
Got it! Should I remove the `transparent_color` parameter in that case or keep it for API consistency?

Reply:
Keep it in case we support multiple colors later.